web/impact/requirements/base.txt

```diff
@@ -8,6 +8,7 @@ hiredis==0.2.0
 markdown # Markdown support for the browsable API.
 mysqlclient
 oauthlib
+newrelic
```

Comment:
I'd consider locking it to the version that accelerate auto-locks it, but not a blocker.

Reply:
Won't fix.